.env.example

```
@@ -15,13 +15,10 @@ OPENAI_API_KEY="your-openai-api-key"
 STRIPE_SECRET_KEY="sk_test_your_stripe_secret_key"
 STRIPE_PUBLISHABLE_KEY="pk_test_your_stripe_publishable_key"
 
-# Chatbase (Legacy - for existing integration)
-NEXT_PUBLIC_CHATBASE_BOT_ID="your-chatbase-bot-id"
-CHATBASE_API_KEY="your-chatbase-api-key"
-CHATBASE_WEBHOOK_SECRET="your-chatbase-webhook-secret"
 
 # AI Agents - Abacus.AI Integration
 ABACUS_AI_API_KEY="your-abacus-ai-api-key"
+NEXT_PUBLIC_ABACUS_API_KEY="your-abacus-api-key"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)